### PR TITLE
Untangle: Update the items on the site ellipsis menu

### PR DESCRIPTION
**Pull request by arthur791004**
Related to https://github.com/Automattic/dotcom-forge/issues/5726

## Proposed Changes

* Update the items on the site ellipsis menu as follows
  * The `Hosting configuration` becomes `Hosting`
  * The `Manage plugins` points to `/wp-admin/plugins.php`
  * The `Performance settings` points to `/wp-admin/options-general.php?page=page-optimize`
  * The `Visit WP Admin` is renamed to `Open WP Admin`

| Site | Before | After |
| - | - | - |
| Simple | image | image |
| Atomic | image | image |

## Testing Instructions

* Go to /sites
* Click the `...` on the site card of your atomic site with wp-admin interface
* Make sure
  * The `Hosting configuration` becomes `Hosting`
  * The `Manage plugins` points to `/wp-admin/plugins.php`
  * The `Performance settings` points to `/wp-admin/options-general.php?page=page-optimize`
  * The `Visit WP Admin` is renamed to `Open WP Admin`

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/sites-dashboard/components/sites-ellipsis-menu.tsx [hunk 1]
```diff
@@ -95,28 +99,36 @@ const SettingsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
...
 	const { __ } = useI18n();
 	const hasManagePluginsFeature =
 		useSelector( ( state ) => siteHasFeature( state, site.ID, WPCOM_FEATURES_MANAGE_PLUGINS ) ) ||
 		isNotAtomicJetpack( site );
 	// If the site can't manage plugins then go to the main plugins page instead
 	// because it shows an upsell message.
 	const [ href, label ] = hasManagePluginsFeature
-		? [ getManagePluginsUrl( site.slug ), __( 'Manage plugins' ) ]
+		? [
+				isWpAdminInterface ? `${ wpAdminUrl }plugins.php` : getManagePluginsUrl( site.slug ),
```
Comment (fushar) [inline client/sites-dashboard/components/sites-ellipsis-menu.tsx:129]: For this one, I think we don't need the `layout/dotcom-nav-redesign` flag check. This should be true for any site with classic view enabled. We're fixing a bug here.

What do you think of putting the logic in the `getManagePluginsUrl()` instead? In the file, we already have a `siteDefaultInterface()` function that checks for the `wpcom_admin_interface` option.

File: client/sites-dashboard/components/sites-ellipsis-menu.tsx [hunk 2]
```diff
@@ -450,7 +473,11 @@ export const SitesEllipsisMenu = ( {
 						<CopySiteItem { ...props } onClick={ startSiteCopy } />
 					) }
 					<MenuItemLink
-						href={ `/settings/performance/${ site.slug }` }
+						href={
+							isWpAdminInterface
```
Comment (fushar) [inline client/sites-dashboard/components/sites-ellipsis-menu.tsx:477]: Similarly, I think we can create a `getPerformanceUrl()` or something and put the logic there (to follow the style)

**Review by fushar — APPROVED** [review]
I think my comments are optional. I'm giving approval for now 👍

File: client/sites-dashboard/components/sites-ellipsis-menu.tsx (hunk 1 above)
Comment (arthur791004) [reply client/sites-dashboard/components/sites-ellipsis-menu.tsx:129]: I'd prefer to keep the current way as I don't want to check whether the interface is wp-admin and the flag is enabled here and there for now 🙈

But I agree that we can refactor it to centralize the logic of getting the label and link of the menu items later 🙂

**Pull request merged**